api/src/npm/tarball.rs

```diff
@@ -133,7 +133,13 @@ pub async fn create_npm_tarball<'a>(
             .get_parsed_source(module.specifier())
             .expect("parsed source should be here");
 
-          let module_info = DefaultModuleAnalyzer::module_info(&source);
+          let analyzer = DefaultModuleAnalyzer;
+
+          let module_info = analyzer.analyze(
+            module.specifier(),
+            source.text_info().text(),
+            js.media_type,
+          )?;
```

Comment:
This will reparse. I think it's now: `ParsedModuleAnalyzer::modulr_info(&source)`